MovingBlocks/Terasology
# chore: screen merge follow-ups

### Contains

* fix screen title
* remove verbose label asking the user to pick a world generator from the drop down
* remove no longer used translation tags
* return user to module selection if no module selected that provides world generator
* differentiate between world generator creation issues and display popup to user if selected world generator cannot be resolved

### How to test

1. Start Terasology
2. Go to advanced game setup
3. Go to universe setup screen
4. Confirm above-mentioned changes

### Outstanding before merging

- [ ] Check and adjust Re-Roll button height to seed text field height
